rasa_core/run.py

```diff
@@ -245,10 +246,13 @@ def load_agent(core_model, interpreter, endpoints,
     _endpoints = AvailableEndpoints.read_endpoints(cmdline_args.endpoints)
     _interpreter = NaturalLanguageInterpreter.create(cmdline_args.nlu,
                                                      _endpoints.nlu)
+    domain = TemplateDomain.load(os.path.join(cmdline_args.core, "domain.yml"))
```

Comment:
this file might not exist: the agent might be loaded from a remote server and this directory might be empty 

It is fine to set the `TrackerStore`'s domain to `None` in line L250 and not load the domain here (the actual domain will be updated once it is loaded) - though we really need to fix this...